storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -0,0 +1,227 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import {
+  Grid,
+  Heading,
+  Image,
+  Link,
+  LinkList,
+  PageHeading,
+  Paragraph,
+  SearchField,
+  Spotlight,
+} from '@amsterdam/design-system-react'
+import {
+  CameraIcon,
+  ChatBubbleIcon,
+  FacebookIcon,
+  LinkedinIcon,
+  TwitterIcon,
+} from '@amsterdam/design-system-react-icons'
+import { Meta, StoryObj } from '@storybook/react'
+import { LinkGroupGridCells } from './LinkGroupGridCells'
+import { NavigationPage } from './NavigationPage'
+import { burgerzakenLinks, parkerenLinks } from './links'
+import { exampleHeading, exampleParagraph } from '../../../components/shared/exampleContent'
+import { commonMeta } from '../common/config'
+
+const meta = {
+  ...commonMeta,
+  title: 'Pages/Amsterdam.nl/Navigation Page',
+  component: NavigationPage,
+} satisfies Meta<typeof NavigationPage>
+
+export default meta
+
+export const Default: StoryObj = {
+  args: {
+    children: (
+      <Grid as="main" gapVertical="large" id="main" paddingBottom="large">
+        <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+          <PageHeading className="ams-mb-m">Burgerzaken</PageHeading>
+          <Paragraph size="large">
+            Heeft u een paspoort, rijbewijs of uittreksel nodig? Of wilt u een verhuizing doorgeven of een geboorte
+            aangeven? Op deze pagina vindt u alle informatie en regelzaken rondom Burgerzaken.
+          </Paragraph>
+        </Grid.Cell>
+        <LinkGroupGridCells linkGroups={burgerzakenLinks} />
+      </Grid>
+    ),
+  },
+}
+
+export const WithInteractiveElement: StoryObj = {
+  args: {
+    children: (
+      <main id="main">
+        <Grid gapVertical="large" paddingBottom="large">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+            <PageHeading className="ams-mb-m">Parkeren</PageHeading>
+            <Paragraph className="ams-mb-m" size="large">
+              {exampleParagraph()}
+            </Paragraph>
+            <Paragraph>{exampleParagraph()}</Paragraph>
+          </Grid.Cell>
+          <LinkGroupGridCells linkGroups={parkerenLinks}></LinkGroupGridCells>
+        </Grid>
+        <Spotlight>
+          <Grid paddingVertical="large">
+            <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+              <Heading className="ams-mb-m" color="inverse" level={2}>
+                Parkeertarieven
+              </Heading>
+              <SearchField className="ams-mb-m">
+                <SearchField.Input placeholder="Zoek op adres" />
+                <SearchField.Button />
```

Comment:
The blue of the button on the purple background kind of hurts my eyes :D

I don't think this is a WCAG failure, since [the button has an icon in it with sufficient contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). Do we want to take clashing colors into account here?

Reply:
We should do something about it, but maybe not in this PR. We’re presenting a page composition here, with our current state of components. Finding things we didn’t anticipate is expected, let’s feed that back to design and go about it as usual.

A white outline might be an option, although the black border now becomes a bit disturbing:

<img width="649" alt="Search Field outline" src="https://github.com/user-attachments/assets/056c87fa-1d53-4155-9c1f-1c1393f2a366" />

Or we could remove the black border altogether on a dark background.

Anyway, out of scope for this PR I would say.